src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -314,13 +320,15 @@ class IpexCommunicationService extends AgentService {
   private async admitIpex(
     notificationD: string,
     holderAidName: string,
-    issuerAid: string,
-    schemaSaid: string
+    issuerAid: string
   ): Promise<void> {
     // @TODO - foconnor: For now this will only work with our test server, we need to find a better way to handle this in production.
-    await Agent.agent.connections.resolveOobi(
-      `${ConfigurationService.env.keri.credentials.testServer.urlInt}/oobi/${schemaSaid}`
-    );
+    for (const schemaSaid of IpexCommunicationService.SCHEMA_SAIDs) {
```

Comment:
No, I mean we need to extract all of the schemas and chained ACDC schemas from the ACDC. Not assume that it will be vLEI, LE, or IIW